lithium/lithium.py

```diff
@@ -154,11 +154,9 @@ def processOptions():
     if args.chunksize:
         minimizeMin = args.chunksize
         minimizeMax = args.chunksize
-        repeat = "last"
```

Comment:
Yeah, the old behavior was to default to "last" when "chunksize" was not used, and to "never" when "chunksize" was used. I used that as a shortcut but maybe it was too confusing.